dev/TabView/TabView.xaml

```diff
@@ -604,9 +619,30 @@
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
 
+                        <Path x:Name="LeftRadiusRender"
+                            IsHitTestVisible="False"
+                            Grid.Column="0"
+                            Canvas.ZIndex="20"
+                            Height="{Binding Source={ThemeResource OverlayCornerRadius},Path=BottomLeft}"
```

Comment:
Why are we using *OverlayCornerRadius* and not *ControlCornerRadius* for this control? Based on the description of OverlayCornerRadius, I find it more intuitive to use *ControlCornerRadius* here. Using OverlayCornerRadius wouldn't have come to my mind to customize the corner radii of TabViewItems (they are not like a popup or flyout which overlay other user created content, for example):

![image](https://user-images.githubusercontent.com/1398851/78139373-b1e04400-7428-11ea-9c3c-62d8cc209364.png)

Reply:
I've used OverlayCornerRadius because all the other tabview corner radii did so. Regarding why we did in the first place, I am not quite sure, but there probably was a reason.

Reply:
Yes, I saw the precendence in the control which is why I assumed you used that one. I'm interested in the initial reason. Perhaps @teaP knows?

Reply:
I would guess Tabs are overlayed on top of the Tab Bar, and it matches the 4px corner radius used in Edge

Reply:
From a developer's/designer's perspective, all I'm seeing is a TabView control which I can add TabViewItems to. There is no button which might display a popup/flyout like a context menu. As such, the "OverlayCorderRadius" resource - based on the above meaning - *is* unexpected here and not intuitive to me as a developer/designer. If the 4px value were chosen for asthetic purposes to match Edgium UI and hence why `OverlayCorderRadius* was used, I would suggest keepign the 4ox corner radius value but changing to a different resource.

Reply:
@Felix-Dev the same page where you got that screenshot from, they explain why we are using the OverlayCornerRadius here:

> ### Corners of flyout and overlay UI elements
> * These can be transient UI elements that appear on screen temporarily, like MenuFlyout, or elements that overlay other UI, like TabView tabs.

https://docs.microsoft.com/en-us/windows/uwp/design/style/rounded-corner

So using OverlayCornerRadius is correct here.

Reply:
@chingucoding Thanks for the link.